index.html

```diff
@@ -1,246 +1,260 @@
+                    <ul class="skill_set text_body_font">
+                        <li class="about_skills"> Languages </li>
+
+                        <li class="li_class"> CSS</li>
+                        <li class="li_class"> Html</li>
+
+                    </ul>
+                    <div class="about_img about_img_1"></div>
                 </div>
-            </section>
+                <div class="about_row about_row_2">
+                    <ul class="skill_set">
+                        <li class="about_skills text_body_font"> Frameworks </li>
 
-            <section  class="about_section">
-                <div class="project_text text_color text_header_font">
-                    <h2>About</h2>
+                        <li class="li_class"> CSS</li>
+                        <li class="li_class"> Html</li>
+                    </ul>
+                    <div class="about_img about_img_2"></div>
                 </div>
-                <div class="text_subject text_color text_body_font about_text">
-                    <p>
-                        Hello!I am a software developer! I can 
-                        help you build a product, feature or website.
-                        Take a look of my works. If you like what you see 
-                        and have a project you need coded, don't hesitate 
-                        and contact me.
-                    </p>
+                <div class="about_row about_row_3">
+                    <ul class="skill_set">
+                        <li class="about_skills text_body_font"> Skills </li>
+                        <li class="li_class"> Github</li>
+                        <li class="li_class"> Terminal</li>
+                    </ul>
+                    <div class="about_img about_img_3"></div>
                 </div>
+            </div>
+
+        </section>
 
-                <a class="text_button text_body_font about_button">
-                    <p> Get my CV</p>
-                    <div class="download_image"></div>  
+        <!-- foooter -->
+        <footer class="contact_section">
+            <div class="mobile_logo"></div>
+
+            <h4 class="title-contact text_header_font">Get Started</h4>
+
+            <p class="section-desc text_body_font">
+                Hello!I am a software developer! I can
+                help you build a product, feature or website.
+                Take a look of my works. If you like what you see
+                and have a project you need coded, don't hesitate
+                and contact me.
+            </p>
+            <form action="https://formspree.io/f/mzbovllg" method="post">
+  <input
+    type="text"
+    name="FullName"
+    placeholder="Enter Your Name"
+    maxlength="30"
+    required/>
+
+    <input type="email" name="Email" placeholder="Enter Your Email" required />
+    <textarea
+        name="Message"
+        placeholder="Enter Your Message"
+        maxlength="500"
+        required>
+        </textarea>

```

Comment:
- [ ] Kindly make the `padding: 60px;` for this input smaller, because the typing be on the center, Kindly fix it.

![image](https://user-images.githubusercontent.com/35535130/160908784-4326c8a4-4905-4395-ac30-51ba266ae63d.png)

Reply:
@keroloussamy, to the best of my knowledge, all the necessary changes have been made.